integration-tests/bats/config.bats

```diff
@@ -287,3 +287,10 @@ teardown() {
     [ "$status" -eq 0 ]
     [[ "$output" =~ "* vegan-btw" ]] || false
 }
+
+@test "config: config doesn't need write permission in current dir" {
+    chmod 111 .
+    run dolt config --list
+    [ "$status" -eq 0 ]
```

Comment:
(minor) when you only need to check that a command executed without returning an error return code, you can skip using `run` and checking `$status` and just call the command directly. If it fails, it'll fail the BATS test. Handy to know for cases like this where you just want to make sure the command doesn't blow up.